website/docs/tutorials/guides/dependencies-management.md

```diff
@@ -0,0 +1,67 @@
+---
+sidebar_label: Dependencies (2 min)
+---
+
+# Dependencies
+
+Protostar manages dependencies (packages) using [git submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules) ([similarly to Foundry](https://onbjerg.github.io/foundry-book/projects/dependencies.html)). This is the reason Protostar expects [git](https://git-scm.com/) to be installed. Keep this in mind, especially if your project uses git submodules.
+
+:::note
+Using Git submodules as a foundation for package management is not an ideal approach. Therefore, [Starkware](https://starkware.co/) plans to create a proper package manager.
```

Comment:
```suggestion
Using Git submodules as a foundation for package management is not an ideal approach. Therefore, [Starkware](https://starkware.co/) plans to create a proper package manager, which we are planning to integrate into protostar in the future.
```

Reply:
Do we?

Reply:
That's a good question 🤔 . Maybe leave it as it was